act-rules-format.md

```diff
@@ -170,16 +170,26 @@ Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em>
 4. the conformance level associated with the accessibility requirement, if one exists.
 
 
-### Outcome Mapping ### {#outcome-mapping}
+### Mapping Conformance Requirements ### {#mapping-conformance-requirements}
+
+An ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying an accessibility requirement for that [=test subject=]. 
+
+A conformance requirement mapped in a rule: 
+
+- is <dfn>not satisfied</dfn> when one or more outcomes for a test target is `failed`, and 
+- can be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>, but cannot be <dfn>not satisfied</dfn> when all of the outcomes are `passed` or `inapplicable` . 
+
+Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
```

Comment:
We need to define what we mean by a satisfying test. I feel like that would be a conformance requirement where `passed` and `inapplicable` outcomes mean the criteria is *satisfied*. If further testing is needed then we can't determine the requirement is *satisfied*

Reply:
I don't think we reference "satisfying tests" in practice. Discuss with @WilcoFiers if term is needed.